lektor/packages.py

```diff
@@ -241,6 +178,12 @@ def list_local_packages(path):
     return rv
 
 
+def hash_directory(path) -> str:
+    """Compute a checksum for a directory."""
+    file_info = FileInfo(None, path)
```

Comment:
FileInfo.checksum may not be exactly what is wanted here.  (FileInfo appears to only compute a checksum over the files in the top-level of a directory.  It does not recurse into subdirectories.)